### ISSUE TITLE
keras实现存在的问题

### ISSUE THREAD

**Issue by 1535966643** [audier/my_deep_project#3]
数据的处理完全是按照作者的方式，就是在构建网络时候我使用keras
model.add(Embedding(input_dim=vocabSize, output_dim=outputCell_size, input_length=time_step))
 model.add(SimpleRNN(units=cellSize, activation='relu', return_sequences=True, dropout=0.2))
 model.add(SimpleRNN(units=64, activation='relu', dropout=0.2))
 model.add(Dense(units=time_step))

这是我的损失函数
optimizers = optimizers.rmsprop(lr=learning_rate)
model.compile(loss='categorical_crossentropy', optimizer=optimizers, metrics=['accuracy'])
我换了损失函数，损失也是一直非常高，怎么解决呢？
